packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.component.js

```diff
@@ -59,6 +60,8 @@ export function AvroRenderer({ colDef, data }) {
 			return (
 				<DefaultValueRenderer
 					value={data.value}
+					isValueOverflown={isValueOverflown}
+					isLongValueToggled={isLongValueToggled}
```

Comment:
Just wondering, instead of passing explicitly those 2 values, would it be better to just spread the rest of the props to each implementation ?
(don't have the answer to that)

Reply:
This component uses those props for some style so we already pick them individually.